staticcheck.conf

```diff
@@ -1 +1,16 @@
-initialisms = ["inherit", "CNI", "IPAM"]
+initialisms = ["ACL", "API", "ASCII",
+	"CNI", "CPU", "CSS",
+	"DNS",
+	"EOF",
+	"GID", "GUID",
+	"HTML", "HTTP", "HTTPS",
+	"ID", "IP", "IPAM", "IPSec",
+	"JSON",
+	"NAT", "NATT",
+	"QPS",
+	"RAM", "RPC",
+	"SLA", "SMTP", "SQL", "SSH",
+	"TCP", "TLS", "TTL",
+	"UDP", "UI", "UID", "URI", "URL", "UTF8", "UUID",
+	"VM",
+	"XML", "XMPP", "XSRF", "XSS"]
```

Comment:
Why are we removing the "inherit" and re-listing the defaults?

```suggestion
initialisms = ["inherit", "CNI", "IPAM", "NAT", "NATT"]
```

Reply:
I missed "inherit" while copy-pasting